DICOM/src/Microsoft.Health.DeID.SharedLib/CryptoHashFunction.cs

```diff
@@ -13,62 +13,61 @@ namespace Microsoft.Health.Dicom.DeID.SharedLib
 {
     public class CryptoHashFunction
     {
-        public static byte[] ComputeHmacSHA256Hash(byte[] input, byte[] hashKey = null)
+        private HMAC _hmac;
+
+        public CryptoHashFunction(byte[] hashKey = null)
         {
-            if (input == null)
+            _hmac = new HMACSHA256();
```

Comment:
Maybe we can put it in else clause to avoid two initializations when a hashKey is given?

Reply:
Thanks, fixed it.